packages/aws-cdk-lib/aws-logs/lib/service-managed-log-group.ts

```diff
@@ -0,0 +1,272 @@
+   * @default "Tags"
+   */
+  readonly responseField?: string;
+  /**
+   * Additional permissions given to the custom resource function to query tags
+   * @default "[]"
+   */
+  readonly permissions?: string[];
+}
+
+/**
+ * A CloudWatch Log Group that is created and managed by another service
+ *
+ * With this construct, the Log Group configuration can be changed anyway.
+ * You must call `bind()` on the ServiceManagedLogGroup, to connect it with the parent resource.
+ *
+ * @resource AWS::Logs::LogGroup
+ */
+export class ServiceManagedLogGroup extends LogGroupBase implements ILogGroup {
+  public readonly logGroupArn: string;
+  public readonly logGroupName: string;
+
+  /**
+   * Tags for the LogGroup.
+   */
+  public readonly tags: cdk.TagManager = new cdk.TagManager(cdk.TagType.KEY_VALUE, 'AWS::Logs::LogGroup');
+
+  private provider: ServiceManagedLogGroupFunction;
+  private props: BaseLogGroupProps;
+  private _logGroupArn?: string;
+
+  constructor(scope: Construct, id: string, props: ServiceManagedLogGroupProps = {}) {
+    super(scope, id);
+
+    this.props = {
+      ...props,
+      retention: validateLogGroupRetention(props.retention),
+    };
+
+    this.logGroupArn = cdk.Lazy.string({
+      produce: () => this._logGroupArn + ':*',
+    });
+    this.logGroupName = cdk.Lazy.string({
+      produce: () => cdk.Stack.of(this).splitArn(this._logGroupArn!, cdk.ArnFormat.COLON_RESOURCE_NAME).resourceName,
+    });
+
+    // Custom resource provider
+    this.provider = this.ensureSingletonProviderFunction();
+  }
+
+  /**
+   * Helper method to ensure that only one instance of the provider function resources is in the stack.
+   * Mimicking the behavior of @aws-cdk/aws-lambda's SingletonFunction to prevent circular dependencies.
+   */
+  private ensureSingletonProviderFunction() {
+    const functionLogicalId = 'ServiceManagedLogGroup' + 'f0360f7393ea41069d5f706d30f37fa7';
+    const existing = cdk.Stack.of(this).node.tryFindChild(functionLogicalId);
+    if (existing) {
+      return existing as ServiceManagedLogGroupFunction;
+    }
+    return new ServiceManagedLogGroupFunction(cdk.Stack.of(this), functionLogicalId);
+  }
+
+  /**
+   * Bind
+   */
+  public bind(options: ServiceManagedLogGroupBindOptions) {
+    this._logGroupArn = options.logGroupArn;
+
+    const resource = new cdk.CfnResource(this, 'Resource', {
+      type: SERVICE_MANAGED_LOG_GROUP_TYPE,
+      properties: {
+        ServiceToken: this.provider.functionArn,
+        LogGroupName: this.logGroupName,
+        LogGroupRegion: cdk.Lazy.string({
+          produce: () => cdk.Stack.of(this).splitArn(this._logGroupArn!, cdk.ArnFormat.COLON_RESOURCE_NAME).region,
+        }),
+        DataProtectionPolicy: this.props.dataProtectionPolicy?._bind(this),
+        KmsKeyId: this.props.encryptionKey?.keyArn,
+        RetentionInDays: this.props.retention,
+        Tags: this.tags.renderedTags,
+        Tagging: options.tagging,
```

Comment:
Is it possible to get tags from the CloudFormation stack? We stopped using tags in our templates and moved to tagging stacks due to some resources not allowing tag updates and/or differences with GovCloud.

Reply:
That's one reason why this PR isn't finished. 😓